<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="messageGroupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>531</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>Message Control</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>22</y>
       <width>511</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="2">
       <widget class="QPushButton" name="renamePushButton">
        <property name="text">
         <string>Rename</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="uploadPushButton">
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="downloadPushButton">
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="descriptionPushButton">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="newFolderPushButton">
        <property name="text">
         <string>New Folder</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="eventPushButton">
        <property name="text">
         <string>Event</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="removePushButton">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QTreeWidget" name="messageTreeWidget">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <column>
         <property name="text">
          <string>File Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Size</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Description</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Duration</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ioBoardGroupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>241</width>
      <height>391</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>IO Board</string>
    </property>
    <widget class="QGroupBox" name="diGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>DI</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>43</width>
        <height>92</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QCheckBox" name="di3CheckBox">
         <property name="text">
          <string>DI3</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="di1CheckBox">
         <property name="text">
          <string>DI1</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="di2CheckBox">
         <property name="text">
          <string>DI2</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="di0CheckBox">
         <property name="text">
          <string>DI0</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="doGroupBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>131</width>
       <height>153</height>
      </rect>
     </property>
     <property name="title">
      <string>DO</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>18</y>
        <width>77</width>
        <height>123</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QCheckBox" name="do0CheckBox">
           <property name="text">
            <string>DO0</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="do1CheckBox">
           <property name="text">
            <string>DO1</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="do2CheckBox">
           <property name="text">
            <string>DO2</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="do3CheckBox">
           <property name="text">
            <string>DO3</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="doManualPushButton">
         <property name="text">
          <string>Manual</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="aiGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>221</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>AI</string>
     </property>
     <widget class="QListView" name="aiListView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="connectGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QGroupBox" name="connMethodGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Type</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>215</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QRadioButton" name="ethernetRadioButton">
         <property name="text">
          <string>ETHERNET</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="ethernetAddrLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>192.168.1.150</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rs485RadioButton">
         <property name="text">
          <string>RS-485</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="rs485AddrLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="connectPushButton">
     <property name="geometry">
      <rect>
       <x>274</x>
       <y>67</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="easDateTimeGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>211</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>EAS Date/Time</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>20</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="2">
        <widget class="QDateTimeEdit" name="dateTimeEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="easGetTimePushButton">
         <property name="text">
          <string>Get</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="easSyncTimePushButton">
         <property name="text">
          <string>Sync</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Connect"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="messageMenu">
    <property name="title">
     <string>&amp;Messages</string>
    </property>
    <addaction name="action_Upload"/>
    <addaction name="action_Download"/>
    <addaction name="action_Rename"/>
    <addaction name="actionRe_move"/>
    <addaction name="actionNew_Folder"/>
    <addaction name="action_Play"/>
    <addaction name="separator"/>
    <addaction name="actionDescription"/>
    <addaction name="actionEvent"/>
   </widget>
   <widget class="QMenu" name="ioBoardMenu">
    <property name="title">
     <string>IO &amp;Board</string>
    </property>
    <addaction name="action_DO"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="messageMenu"/>
   <addaction name="ioBoardMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Connect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Upload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Upload</string>
   </property>
  </action>
  <action name="action_Download">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Download</string>
   </property>
  </action>
  <action name="action_Rename">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Rename</string>
   </property>
  </action>
  <action name="actionRe_move">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Re&amp;move</string>
   </property>
  </action>
  <action name="action_Play">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Play</string>
   </property>
  </action>
  <action name="action_DO">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;DO</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionNew_Folder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New Folder</string>
   </property>
  </action>
  <action name="actionDescription">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </action>
  <action name="actionEvent">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Event</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
